opentelemetry-user-events-logs/examples/basic-logs.rs

```diff
@@ -14,8 +15,10 @@ fn init_logger() -> SdkLoggerProvider {
     let fmt_layer = tracing_subscriber::fmt::layer().with_filter(filter_fmt);
     let _guard = tracing_subscriber::registry().with(fmt_layer).set_default(); // Temporary subscriber active for this function
 
-    SdkLoggerProvider::builder()
-        .with_user_events_exporter("myprovider")
+    let user_event_options = ExportOptions::builder("myprovider").build().unwrap();
```

Comment:
Let's use either `export_options` or `user_event_export_options` as the variable name.

```suggestion
    let export_options = ExportOptions::builder("myprovider").build().unwrap();
```